charts/rancher-gke-operator/1.0.300/values.yaml

```diff
@@ -4,7 +4,7 @@ global:
 gkeOperator:
   image:
     repository: rancher/gke-operator
-    tag: v1.1.1-rc7
+    tag: v1.0.3
 
 httpProxy: ""
 httpsProxy: ""
```

Comment:
I think 1.0.300 should have more changes no ? Looks like questions.yaml is still set on old one: https://github.com/rancher/system-charts/blob/be12cf837e206bd3b8917b5cb3ea626f103427e0/charts/rancher-gke-operator/1.0.300/questions.yaml